app/controllers/accounting/master_finances_controller.rb

```diff
@@ -67,6 +68,12 @@ def destroy
 
     private
 
+      def balance_total
+        if @master_finance.done?
+          @master_finance.total === calc_balance(@master_finance, @master_finance.balance)
+        end
+      end
```

Comment:
Acho que isso ainda a errado.

Primeiro que você quer atribuir o valor, e aqui você ta usando `===`, que é para comparar.
Segundo, você ta fazendo isso no index e no show, acho que o certo seria fazer algo no create ou update, para gravar o valor, e no index e show, vc só le do model.

Reply:
é verdade, confundi aí os sinais. arrumei e coloquei no edit, create, update e destroy...
mas agora ele dá erro pois não reconhece esse método calc_balance que é um helper...
undefined method `calc_balance' for #<Accounting::MasterFinancesController:0x00007f0f82b569e8>